app/facades/merchants_facade.rb

```diff
@@ -0,0 +1,31 @@
+class MerchantsFacade
+  def self.find(name)
+    results = MerchantService.find_all(name)
+
+    if results != nil
+        results.map do |result|
+          Merchant.new(result)
+        end
+    end
```

Comment:
Another edge case you could test here is that if the data root key came back, that the array it was pointing to wasn't empty.